### ISSUE TITLE
Add other block ref options to context menu... Send Block Ref to DNP ... Send Block Ref to a Page

### ISSUE THREAD

**Issue by GitMurf** [mgmeyers/obsidian-copy-block-link#2]
Another fun thing I was thinking about (wrote above) was:

- Right-Click > Send Block ref to DNP
- Right-Click > Send block ref to a page (pop up chooser)

This would be a quick way to aggregate a bunch of block refs quickly when reviewing notes, and send them somewhere in one click.

**Comment by wenlzhang** [comment]
Have been expecting something like this for a long time. It would be really great if features like these can be supported!

Just want to add some thoughts on

- Send to daily/weekly/monthly notes, which are supported by the Periodic Notes plugin ([liamcain/obsidian-periodic-notes: Create/manage your daily, weekly, and monthly notes in Obsidian](https://github.com/liamcain/obsidian-periodic-notes))
  - Does DNP refer to this?
  - Maybe one can use the Natural Language Dates plugin to input specific dates to send to
    - [argenos/nldates-obsidian: Work with dates in natural language in Obsidian](https://github.com/argenos/nldates-obsidian)
  - Or maybe one can bring up a pop-up window for selecting the Month/Week/Day to send to
    - [TfTHacker/obsidian42-jump-to-date: Jump to a date via a convenient popup](https://github.com/TfTHacker/obsidian42-jump-to-date)
- Send to any notes/pages by using a window like the one for opening notes from Command Palette. This can be very useful in the following case:
  - One writes/improves a note and thinks that the current note can be linked to another note.
  - One can then send the link of the current block/header/note to the desired note.
    - It would also be nice if one can add some quick notes together with the link, for better reviewing later.
    - Or there can be an option to send the link to the desired note and open that note in a new pane, where one can also add some quick notes, e.g., how these two notes are connected.
  - One check links to the current note in the link pane, sort related notes based on modification time, and go to the desired note to further improve it.

In short, having the proposed features supported would greatly improve the workflow for reviewing/summarizing/improving notes.
